staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2862,6 +2862,31 @@ type PodDNSConfigOption struct {
 	Value *string `json:"value,omitempty" protobuf:"bytes,2,opt,name=value"`
 }
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// EphemeralContainer describes a container to attach to a running pod for troubleshooting.
+// This field is alpha-level and is only honored by servers that enable the DebugContainers feature.
+type EphemeralContainer struct {
+	metav1.TypeMeta `json:",inline"`
```

Comment:
This is a subresource, right? It should probably have a metadata field (see e.g. scale: https://github.com/verb/kubernetes/blob/101ade567068928ba4ac106b66b5d6173f230d7a/staging/src/k8s.io/api/extensions/v1beta1/types.go#L62)